lms/templates/courseware/progress.html

```diff
@@ -50,23 +50,25 @@
         <h1>${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email)}</h1>
       </header>
 
-      %if settings.FEATURES.get("SHOW_PROGRESS_SUCCESS_BUTTON"):
```

Comment:
I'll follow up with one of our partners who I know is using this to make sure it's okay to remove it.  If so, we'll also need to remove references to this feature flag in the settings files.